spec/factories/books.rb

```diff
@@ -8,8 +8,8 @@
     editor { Faker::Lorem.words(2).join(' ') }
     google_books_id { Faker::Lorem.characters(10) }
     introducer { Faker::Lorem.words(2).join(' ') }
-    isbn_10 '0140446044'
-    isbn_13 '9780140446043'
+    isbn_10 '0804720991'
+    isbn_13 '9780804720991'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.